environments/prod/helm/orchestration-workflows/clinvar/values.yaml

```diff
@@ -0,0 +1,20 @@
+enable: false
+argo:
+  env: prod
+  vaultPrefix: secret/dsde/monster/prod/command-center
+  namespace: clinvar
+  artifactBucket: broad-dsp-monster-clingen-prod-argo-archive
+chart:
+  git: false
+  ref: 1.0.0-RC1
```

Comment:
Version explanation: 1st release candidate for version 1. Release candidate == could plausibly be re-tagged as the final version and redeployed with no changes as the "official" release.